Shared/Samples/Navigate route/NavigateRouteView.swift

```diff
@@ -0,0 +1,354 @@
+        func solveRoute() async {
+            do {
+                // Creates the default parameters from the route task.
+                let parameters = try await routeTask.makeDefaultParameters()
+                
+                // Configures the parameters.
+                parameters.returnsDirections = true
+                parameters.returnsStops = true
+                parameters.returnsRoutes = true
+                parameters.outputSpatialReference = .wgs84
+                
+                // Sets the stops on the parameters.
+                parameters.setStops(stops)
+                
+                // Solves the route based on the parameters.
+                routeResult = try await routeTask.solveRoute(using: parameters)
+                
+                // Enables the navigate button.
+                isNavigateDisabled = false
+            } catch {
+                self.error = error
+            }
+        }
+        
+        /// Sets the route tracker and location display's data source with a solved route. Updates
+        /// the list of directions, the route ahead graphic, and the map's viewpoint.
+        func setNavigation() {
+            guard let routeResult else { return }
+            
+            // Creates a route tracker from the route results.
+            routeTracker = RouteTracker(routeResult: routeResult, routeIndex: 0, skipsCoincidentStops: true)
+            guard let routeTracker else { return }
+            
+            routeTracker.voiceGuidanceUnitSystem = Locale.current.usesMetricSystem ? .metric : .imperial
+            
+            // Gets the route and its geometry from the route result.
+            guard let firstRoute = routeResult.routes.first,
+                  let routeGeometry = firstRoute.geometry else {
+                return
+            }
+            
+            // Updates the directions.
+            directions = firstRoute.directionManeuvers
+            
+            // Creates the mock data source from the route's geometry.
+            let densifiedRoute = GeometryEngine.geodeticDensify(
+                routeGeometry,
+                maxSegmentLength: 50,
+                lengthUnit: .meters,
+                curveType: .geodesic
+            ) as! Polyline
+            let mockDataSource = SimulatedLocationDataSource(polyline: densifiedRoute)
+            
+            // Creates a route tracker location data source.
+            let routeTrackerLocationDataSource = RouteTrackerLocationDataSource(
+                routeTracker: routeTracker,
+                locationDataSource: mockDataSource
+            )
+            
+            // Sets the location display's data source.
+            locationDisplay.dataSource = routeTrackerLocationDataSource
+            
+            // Updates the graphics and viewpoint.
+            routeRemainingGraphic.geometry = routeGeometry
+            viewpoint = Viewpoint(boundingGeometry: routeGeometry)
```

Comment:
```suggestion
            viewpoint = Viewpoint(boundingGeometry: routeGeometry, rotation: 0)
```
Like to see the map rotate back to the initial state.